tensorflow/compiler/xla/service/hlo_instruction.cc

```diff
@@ -2538,6 +2550,9 @@ HloInstructionProto HloInstruction::ToProto() const {
       proto.mutable_outer_dimension_partitions()->Add(idx);
     }
   }
+  if (!frontend_attributes_.map().empty()) {
```

Comment:
Why do we need to check `!frontend_attributes_.map().empty()`?

Reply:
Done